pulp_python/app/tasks/publish.py

```diff
@@ -118,6 +118,7 @@ def write_project_page(name, simple_dir, package_releases, publication):
     metadata_relative_path = f"{project_dir}index.html"
 
     with open(metadata_relative_path, "w") as simple_metadata:
+        # todo?
```

Comment:
Let's leave this for the future. We might not implement the feature for publications if we are planning to remove them for Pulp 4.

Reply:
No, it's not deprecated in the sense that we will still fix issues with it, but we will point users to use the new preferred method if they want all the latest features.